src/index.html

```diff
@@ -15,8 +15,28 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+
   </head>
   <body>
     <h1>Moyo header</h1>
+     <nav>
+       <header class="conteiner">
+         <img src="src/images/logo.png" alt="Logotip" class="logo">
+         <ul>
+           <li><a href="#sectionnpm run lint" class="apple text">Apple</a></li>
```

Comment:
There seems to be a typo in the href attribute of the first list item link. The value '#sectionnpm run lint' is likely incorrect and should be corrected to a valid section ID or URL, such as '#section'.